index.html

```diff
@@ -5,8 +5,75 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="description" content="Victor's Portfolio - Web Developer" />
+    <link rel="stylesheet" href="./styles.css" />
     <link rel="shortcut icon" href="./assets/favicon.png" type="image/x-icon" />
     <title>My portafolio</title>
   </head>
-  <body></body>
+  <body>
+    <header class="toolbar">
+      <div class="myLogo">My Logo</div>
+      <div class="nav">
```

Comment:
- [ ] Since this element will be containing the navigation menu for your portfolio, it's recommended to use a `<nav>` tag for semantic purposes.